recipes-bsp/u-boot/u-boot-socfpga-common.inc

```diff
@@ -9,3 +9,8 @@ PV_append = "+git${SRCPV}"
 SRC_URI = "git://git.denx.de/u-boot.git;branch=master"
 
 S = "${WORKDIR}/git"
+
+PROVIDES += "u-boot"                                                            
+PKG_${PN} = "u-boot"                                                            
+PKG_${PN}-dev = "u-boot-dev"                                                    
+PKG_${PN}-dbg = "u-boot-dbg"
```

Comment:
i will switch to rprovidesand remove the PKG junk.  I just copied that from another recipe that was doing what i needed.